src/base/rss/rss_article.cpp

```diff
@@ -104,6 +106,10 @@ bool Article::isRead() const
     return m_isRead;
 }
 
+QString Article::value(const QString key) const {
+    return m_varHash.value(key, QString('_')).toString();
```

Comment:
@glassez, @ATGardner:  Also, some filesystems do not handle spaces very well, often requiring additional quotes, slashes/backslashes, and the like.